virtualizarr/tests/test_backend.py

```diff
@@ -320,46 +327,66 @@ def test_virtualizarr_vs_local_nisar(self, hdf_backend):
         xrt.assert_equal(dsXR, dsV)
 
 
-@requires_kerchunk
-def test_open_empty_group(empty_netcdf4_file):
-    vds = open_virtual_dataset(empty_netcdf4_file, indexes={})
-    assert isinstance(vds, xr.Dataset)
-    expected = Dataset()
-    xrt.assert_identical(vds, expected)
-
-
-@requires_kerchunk
+@parametrize_over_hdf_backends
 class TestOpenVirtualDatasetHDFGroup:
-    def test_open_subgroup(self, netcdf4_file_with_data_in_multiple_groups):
+    def test_open_empty_group(self, empty_netcdf4_file, hdf_backend):
+        vds = open_virtual_dataset(empty_netcdf4_file, indexes={}, backend=hdf_backend)
+        assert isinstance(vds, xr.Dataset)
+        expected = Dataset()
+        xrt.assert_identical(vds, expected)
+
+    def test_open_subgroup(
+        self, netcdf4_file_with_data_in_multiple_groups, hdf_backend
+    ):
         vds = open_virtual_dataset(
-            netcdf4_file_with_data_in_multiple_groups, group="subgroup", indexes={}
+            netcdf4_file_with_data_in_multiple_groups,
+            group="subgroup",
+            indexes={},
+            backend=hdf_backend,
+        )
+        # This should just be ["bar"] see #401 for more information
+        assert list(vds.variables) == (
+            ["bar", "dim_0"] if hdf_backend == HDFVirtualBackend else ["bar"]
         )
-        assert list(vds.variables) == ["bar"]
         assert isinstance(vds["bar"].data, ManifestArray)
         assert vds["bar"].shape == (2,)
 
-    def test_open_root_group_manually(self, netcdf4_file_with_data_in_multiple_groups):
+    def test_open_root_group_manually(
```

Comment:
Couldn't this test be combined with the one below by parameterizing `group` over `("", None)`?

Reply:
Done in https://github.com/zarr-developers/VirtualiZarr/pull/395/commits/8706af51c340898d29b5041dc3bc265c3b868b5e